auditlog/models.py

```diff
@@ -480,12 +479,28 @@ def changes_display_dict(self):
                         value = f"{value[:140]}..."
 
                     values_display.append(value)
-            verbose_name = model_fields["mapping_fields"].get(
-                field.name, getattr(field, "verbose_name", field.name)
-            )
+            verbose_name = self.verbose_name(field_name)
             changes_display_dict[verbose_name] = values_display
         return changes_display_dict
 
+    def verbose_name(self, field_name: str):
```

Comment:
I would say let's just move the logic to `verbose_name` and reuse it in `mixins.py` and do not change it.

Reply:
alright then. if that's what you prefer!